policyengine_us/parameters/gov/states/mt/tax/income/credits/rebate/property/amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: Montana provides the following property tax rebate amount.
+values:
+  2022-01-01: 675
+metadata:
+  period: year
+  unit: currency-USD
+  label: Montana property tax rebate amount
+  reference:
+    - title: 2023 Montana Form 2 Individual Income Tax Instructions, What's New?, Property Tax Rebates
```

Comment:
could you add the 2022 and 2023 tax forms here? 
If the value changes between 2022 and 2023 in the tax forms, we should show that

Reply:
i added 2022 property tax rebate form, i wasn't able to find one for 2023 but the form 2 instructions mention the same rate for 2022 and 2023 and online also mentions the same 675 value